ES/ESmy015.xml

```diff
@@ -354,7 +354,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           Rounded letter shapes. The top of ጥ is flat (e.g. <locus target="#28ra"/>,
                             <locus target="#33ra"/>); the left leg of ለ is leant upon the right leg. The heads of ይ and ደ
                           extend downwards touching the foundation of the letter. The hand is similar to <ref target="#h1">Hand 1</ref>
-                          of <ref type="mss" corresp="ESsmm001">ESsmm001</ref> and <ref type="mss" corresp="ESamq011">ESamq011</ref>.
+                          of <ref type="mss" corresp="ESamq011">ESamq011</ref>.
```

Comment:
the text inside of the tag should be normal ID i think
`<ref type="mss" corresp="ESamq011">AMQ-011</ref>`